spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done do
+      status { "done" }
```

Comment:
enumで定義した値はStringではなくSymbolで記載しましょう。これにより、該当のカラムがenumを使用していることが伝わります。
```suggestion
      status { :done }
```

Reply:
修正しました。